src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -21,7 +18,7 @@ export default class extends Controller {
 
         headerTargets.forEach((header) => {
             const isOpen = header.getAttribute('aria-expanded') === 'true';
-            const content = header.nextElementSibling;
+            const subtables = this.getNextSiblingsWithClass(header, 'whatwedo_table-subtable');
```

Comment:
What are all these subtable references? Where is this coming? It's not clear what this is. Is this a real HTML table? Why would it expand?

(answer by updating documentation)

Reply:
@renestalder I adjusted the commit message to give more context. But subTables are a functionality of araise (not documented yet or not that I could find it)

Reply:
@marcwieland95 

> But subTables are a functionality of araise

That's not enough for this to work without context. Also that it is only in the commit message. We clearly should have a couple of code comments here to make this understandable without knowing every part of araise. A clear explanation of where this is coming from and why handle this thing called "subtable" in this controller here.

Reply:
Code comments done and const renamed to `contents`. Since an accordion normally included a header and content part.

@renestalder What annoys me in that code is that there's a lot of duplicated code. Can parts be shared between `initialize` and `toggle`?

Reply:
@renestalder 
Okay. That sound better indeed and after looking at the code again makes much more sense. Will refactor that.

`initalize` seems better in this case as it sets an initial value.
But when a controller is disconnected, so `connect` and `initalize` actually make a difference, I have no clue.

Reply:
@marcwieland95

Looking at the current change, I don't think there is a reasoning having to `apply` functions for both, clsoing and opening. My proposal rather went into the direction, having separate closing and opening functions with one apply-function that calls the one or other depending on the current state. I believe, looking at the code, this still could be true. But its a "Nit" now.